src/js/routes/More/Elections.jsx

```diff
@@ -33,38 +32,37 @@ export default class Elections extends Component {
   onElectionStoreChange () {
     const electionsList = ElectionStore.getElectionList();
     const electionsLocationsList = [];
-    let voter_ballot; // A different format for much of the same data
+    let voterBallot; // A different format for much of the same data
     const voterBallotList = [];
-    let one_ballot_location;
-    let ballot_location_shortcut;
-    let ballot_returned_we_vote_id;
+    let oneBallotlocation;
+    let ballotLocationshortcut;
+    let ballotReturnedweVoteid;
```

Comment:
"oneBallotlocation" should be "oneBallotLocation"
"ballotLocationshortcut" should be "ballotLocationShortcut"
"ballotReturnedweVoteid" should be "ballotReturnedWeVoteId"